test/e2e/instrumentation/logging/stackdrvier/utils.go

```diff
@@ -57,6 +57,7 @@ const (
 	podsScope logProviderScope = iota
 	eventsScope
 	systemScope
+	stackdriverSinkCountLimit = 90
```

Comment:
Please move this to consts section above and add a comment.

Reply:
done